### ISSUE TITLE
please create a policy around not renaming release tarballs in the archive repo

### ISSUE THREAD

**Issue by cmpunches** [libexpat/libexpat#460]
This breaks automation.  I realize that there was a vulnerability that was concerning, there; This is a request to instead put some kind of lock flag in the makefile that will tell the user to use the next version up or something?

Here is an example of the type of impact it has:
https://github.com/crosstool-ng/crosstool-ng/issues/1479

A 404 is probably not the visibility/reporting that was hoped for there.

**Comment by hartwork** [comment]
Hi @cmpunches,

thanks for bringing this topic up. I take responsibility for the renames.

I noticed that a large part of the internet and the industry fails at updating to versions of Expat with the vulnerabilities fixed. The situation is that (1) everything before 2.2.8 is known vulnerable in particular, (2) we spend quite a bit of time upstream fixing vulnerabilities and for a reason, (3) at Source Forge alone (where release archives are still mirrored) there have been sometime ~300 downloads per single vulnerable version per week. That's something that I am rather unhappy about.

So the idea was to (at least temporarily) trigger some 404s explicitly, to make people wake up and remind them of the old vulnerable Expat they forgot about, update their versions, and ideally move most of those downloads to more recent versions, at least to 2.2.8+.  I did expect one or two people mailing me explaining that they do have all CVE-fixes backported on top and have a deliberate reason and ask to rename one particular file back.

The example you share is for version 2.2.6. Why is crosstool-ng using vulnerable libexpat in 2021?

Best, Sebastian
